packages/viewer/src/interactions/baseInteractionHandler.ts

```diff
@@ -141,13 +140,12 @@ export abstract class BaseInteractionHandler implements InteractionHandler {
       !this.isDragging
     ) {
       this.beginDrag(event);
-      didBeginDrag = true;
       this.isDragging = true;
     }
 
     // We only invoke drag interactions for mouse events after a beginDrag has
     // been invoked.
```

Comment:
is this comment inaccurate now?

Reply:
Yeah, it looks like it will be. I'll remove it